src/lib/modernExtend.ts

```diff
@@ -312,17 +320,32 @@ export function enumLookup(args: EnumLookupArgs): ModernExtend {
         },
     }];
 
-    return {exposes: [expose], fromZigbee, toZigbee, isModernExtend: true};
+    const configure: Configure = async (device, coordinatorEndpoint, logger) => {
+        // call setupAttributes if we have a configureReporting payload or configureSkipRead is not set, but then use readOnly=true
+        const ep = device.getEndpoint(isNumber(configureEndpointId) ? configureEndpointId : 1);
+        const reportConfig = (configureReporting !== undefined) ? {...configureReporting, ...{attribute: attribute}} : {attribute: attribute};
+        if ((configureReporting !== undefined) || !configureSkipRead) {
+            await setupAttributes(ep, coordinatorEndpoint, cluster.toString(), [reportConfig], logger, (configureReporting === undefined));
+        }
+    };
+
+    return {exposes: [expose], fromZigbee, toZigbee, configure, isModernExtend: true};
 }
 
 export interface NumericArgs {
-    name: string, cluster: string | number, attribute: string | {id: number, type: number}, description: string,
-    zigbeeCommandOptions?: {manufacturerCode: number}, readOnly?: boolean, unit?: string, endpoint?: string,
+    name: string, cluster: string | number, attribute: string | {ID: number, type: number}, description: string,
+    zigbeeCommandOptions?: {manufacturerCode?: number, disableDefaultResponse?: boolean}, readOnly?: boolean, unit?: string,
+    endpoint?: string, configureEndpointId?: number, configureReporting?: Partial<ZHConfigureReportingItem>, configureSkipRead?: boolean,
     valueMin?: number, valueMax?: number, valueStep?: number, scale?: number,
 }
 export function numeric(args: NumericArgs): ModernExtend {
-    const {name, cluster, attribute, description, zigbeeCommandOptions, unit, readOnly, valueMax, valueMin, valueStep, endpoint, scale} = args;
-    const attributeKey = isString(attribute) ? attribute : attribute.id;
+    const {
+        name, cluster, attribute, description,
+        zigbeeCommandOptions, readOnly, unit, endpoint,
+        configureEndpointId, configureReporting, configureSkipRead,
```

Comment:
The `configureSkipRead` option doesn't seem to be used anywhere, if that's the case I propose to leave it out for now.

Reply:
I added it in case you can't read an attribute. I can drop it again.